NEXT_CHANGELOG.md

```diff
@@ -26,6 +26,87 @@ By [@USERNAME](https://github.com/USERNAME) in https://github.com/apollographql/
 # [x.x.x] (unreleased) - 2022-mm-dd
 
 ## ❗ BREAKING ❗
+
+### Remove support for `rhai.input_file` and support extra configmaps ([PR #XXXX](https://github.com/apollographql/router/issues/XXXX))
+
+The existing `rhai.input_file` mechanism doesn't really work for most helm use cases. This PR removes this mechanism and introduces an alternative method for making rhai scripts available to containerised routers.
+
+By adding support for extraConfigmaps to the router helm chart,  a very convenient mechanism for adding support for rhai scripts is provided.
+
+Example: Create a configmap which contains your rhai scripts.
+
+```yaml
```

Comment:
I was thinking instead of them creating a configmap they could possibly inline the config by using the tpl render function that the extraConfigMaps uses.

Reply:
Of course, still have an option for existing configmap tho.

Reply:
Ok. I've added a comment to the PR you linked in the issue. I'm keen to avoid the requirement for `rhai` specific config. I'd like us to be able to just use "standard" extension mechanisms for doing this (Volumes, ConfigMaps)